cartesian_impedance_msgs/msg/MaxCartesianVelocity.msg

```diff
@@ -0,0 +1,11 @@
+# parameters for Maximum allowed Cartesian Velocity when in Cartesian Impedance
+# the values are divided into translational and rotational
+# each of the translational and rotational components is defined as Vector3
+# for more info on Vector3 messages please see the following link:
+# http://docs.ros.org/api/geometry_msgs/html/msg/Vector3.html
+
+# trans: [mm/s]
```

Comment:
Here gets little bit inconsistent I know, it is my bad since I added these units according to the available robot documentation I have. On the iiwa these units are in mm/s, which is inconsistent to all their previous units... If you agree I think we should use everywhere meters as units and convert this accordingly to specific robot definitions inside the controller code.